test/integration/pull_int_test.go

```diff
@@ -98,15 +98,15 @@ func (suite *PullIntegrationTestSuite) TestMergeBuildScript() {
 	proj, err := project.FromPath(ts.Dirs.Work)
 	suite.NoError(err, "Error loading project")
 
-	_, err = buildscript.NewScriptFromProject(proj, nil)
+	_, err = buildscript.ScriptFromProjectWithFallback(proj, nil)
```

Comment:
Shouldn't this be without the fallback to verify the buildscript was created? Otherwise, this call creates it and the next line will always succeed.

Reply:
That's probably true, but this PR isn't about fixing all the buildscript things, just the `state commit` one. This code is still functionally the same as before, the function just got renamed.